### ISSUE TITLE
Question - Potential Conflict on Decky Installation

### ISSUE THREAD

**Issue by aggietallboy** [moraroy/NonSteamLaunchers-On-Steam-Deck#344]
Which should we be using? Desktop installed from here, or Decky Plugin (are they the same?)

Decky gets stuck for a VERY long time on second library for me.

**Comment by moraroy** [comment]
@aggietallboy yeah they are the same but, send me your log to see why it gets stuck

**Comment by aggietallboy** [comment]
Apolgies -- super noob on steamdeck, just got on Friday -- where are the logs?

**Comment by moraroy** [comment]
@aggietallboy its in the Downloads folder for the NonSteamLaunchers log and then for the decky plugin I think it's like home/deck/homewbrew/plugins/logs/NonSteamLaunchers or something like that
